test/specs/commonTests.js

```diff
@@ -304,10 +304,18 @@ export const isConformant = (Component, options = {}) => {
           'You may need to hoist your event handlers up to the root element.\n'
         )
 
+        let expectedArgs = [eventShape]
+        let errorMessage = 'was not called with (event)'
+
+        if (_.has(Component.propTypes, listenerName)) {
+          expectedArgs = [eventShape, props]
```

Comment:
I see all tests are passing, but how does this work for things like the Input onChange where the `data` arg's `value` key != to the `props` value key?  Same for Checkbox's `checked` prop.